src/main/scala/nl.knaw.dans.easy.solr4files/components/Solr.scala

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.io.File
+import java.net.URL
+
+import nl.knaw.dans.easy.solr4files.{ FeedBackMessage, SolrLiterals }
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import org.apache.solr.client.solrj.SolrRequest.METHOD
+import org.apache.solr.client.solrj.impl.HttpSolrClient
+import org.apache.solr.client.solrj.request.ContentStreamUpdateRequest
+import org.apache.solr.client.solrj.{ SolrClient, SolrQuery }
+import org.apache.solr.common.util.ContentStreamBase
+
+import scala.util.Try
+
+trait Solr {
+  this: DebugEnhancedLogging =>
```

Comment:
use extends here

Reply:
Why? its done like that in other situations.

Reply:
when you compose the traits, you need to add DebugEnhancedLogging there as well. Also, this is not something you want to swap out for other (sub)types of logging. That's what @janvanmansum and I agreed upon.